app/test/fixtures/reports_data.py

```diff
@@ -0,0 +1,298 @@
+import pytest
+
+__orders = [
+    {
+        "_id": 1,
+        "client_address": "fake address",
+        "client_dni": "01234567891",
+        "client_name": "Christopher Sandoval",
+        "client_phone": "123456789",
+        "date": "2022-03-02T19:15:04",
+        "detail": [
+            {
+                "_id": 1,
+                "beverage": None,
+                "ingredient": {
```

Comment:
'Green peppers' and 'tomato' are being ordered both 4 times, which does not make it very clear which is the most requested ingredient for the test of the most requested ingredient.